lib/Payout.php

```diff
@@ -33,6 +33,11 @@ class Payout extends ApiResource
 
     const OBJECT_NAME = "payout";
 
+    use ApiOperations\All;
```

Comment:
Put all of the use statements after OBJECT_NAME in all places where this wasn't the case.